library/xml-model/src/test/java/org/cru/godtools/xml/model/TextTest.kt

```diff
@@ -25,4 +28,36 @@ class TextTest {
         assertEquals(Color.GREEN, text2.textColor)
         assertEquals(Text.Align.END, text2.textAlign)
     }
+
+    @Test
+    fun testTextStartImage() {
+        val resource = mock<Resource>()
+        val manifest = spy(Manifest()) { on { getResource("image.png") } doReturn resource }
+        val text = Text(manifest, "text", startImage = "image.png", startImageSize = 65)
+        assertEquals(resource, text.startImage)
+        assertEquals(65, text.startImageSize)
+    }
+
+    @Test
+    fun testTextStartImageNull() {
+        val text = Text(mockManifest(), "text", startImage = null)
+        assertEquals(null, text.startImage)
+        assertEquals(40, text.startImageSize)
```

Comment:
you should use `DEFAULT_IMAGE_SIZE` here instead of `40`

Reply:
you will need to make `DEFAULT_IMAGE_SIZE` `internal` and add a `@VisibleForTesting` annotation to indicate that the increased visibility is only for testing purposes